### PR TITLE
docs(README): fix two-factor async function mistype

### PR DESCRIPTION
**Pull request by TimDurward**
Async function for Authenticating via two-factor auth should be `async on2fa()` not `async on2Fa`.

Related discussion: https://github.com/octokit/rest.js/issues/1207#issuecomment-457098225

-----
[View rendered README.md](https://github.com/TimDurward/rest.js/blob/master/README.md)

### CODE REVIEW

**Comment by gr2m** [comment]
I made the same typo in the typescript definitions 🤦‍♂️ could you fix that, too?
https://github.com/octokit/rest.js/blob/b50a1acb2a1e5f0baf03311022ae3c2a7bb3b478/scripts/templates/index.d.ts.tpl#L46

File: README.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ The `auth` option can be
      auth: {
        username: 'octocat',
        password: 'secret',
-       async on2Fa () {
+       async on2fa () {
```
[the remark this answers is not in the document]
Comment (TimDurward) [reply README.md:155]: Nice catch! Sure thing.

**Comment by TimDurward** [comment]
@gr2m Committed changes in regards to your review.

**Review by gr2m — APPROVED** [review]
Perfect, thanks a lot!

**Pull request merged**